subxt/src/client.rs

```diff
@@ -169,6 +169,28 @@ impl<T: Config> Client<T> {
         StorageClient::new(&self.rpc, &self.metadata, self.iter_page_size)
     }
 
+    /// Convert the client to a runtime api wrapper for custom runtime access.
+    ///
+    /// The `subxt` proc macro will provide methods to submit extrinsics and read storage specific
+    /// to the target runtime.
+    pub fn to_runtime_api<R: TryFrom<Self>>(
+        self,
+    ) -> Result<R, <R as TryFrom<Client<T>>>::Error> {
+        self.try_into()
+    }
+
+    /// Skip metadata validation during conversion to the Runtime API.
+    pub fn skip_metadata_validation(self) -> ClientUnchecked<T> {
```

Comment:
Aaah, this is what I was looking for; the ability to skip the whole-metadata validation :)